--- less-436/less.nro.manpage	2009-07-08 00:27:21.000000000 +0200
+++ less-436/less.nro	2010-01-04 14:45:16.979806602 +0100
@@ -1290,7 +1290,6 @@ The remainder of LESSBINFMT is a string 
 printf-style escape sequence (a % followed by x, X, o, d, etc.).
 For example, if LESSBINFMT is "*u[%x]", binary characters
 are displayed in underlined hexadecimal surrounded by brackets.
-The default if no LESSBINFMT is specified is "*s<%X>".
 The default if no LESSBINFMT is specified is "*s<%02X>".
 Warning: the result of expanding the character via LESSBINFMT must
 be less than 31 characters.
